scripts/helm_deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I am putting scripts here for clarity of what it's happening, but I expected these to be subsumed by Jenkins libraries steps and being progressively removed from here.